lib/rentlinx.rb

```diff
@@ -2,9 +2,41 @@
 
 # The Rentlinx module provides the means to interact with the Rentlinx API.
 module Rentlinx
+  @@username = nil
```

Comment:
I don't see any reason to support that at this point.  We can add it down the road if it's needed.